### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Argzf** [domcloud/domcloud#42] [Q&A]
Hello there,
Today I’ve tried to go to my website but it keeps telling me that it is not secure and when I press open anyways it doesn't open it and I don't know what to do

My subdomain: `http://gzf-linktred.domcloud.io/`

**Comment by willnode** [comment]
You can [ask for validating SSL again](https://domcloud.co/docs/features/runner#ssl) after changing the domain name.

Go to _deploy_ tab and run this command:

```
features:
- ssl
```
